src/profiler.cpp

```diff
@@ -340,9 +340,10 @@ int Profiler::getNativeTrace(void* ucontext, ASGCT_CallFrame* frames, EventType
 int Profiler::convertNativeTrace(int native_frames, const void** callchain, ASGCT_CallFrame* frames, EventType event_type) {
     int depth = 0;
     jmethodID prev_method = NULL;
+    const void* start_addr;
 
     for (int i = 0; i < native_frames; i++) {
-        const char* current_method_name = findNativeMethod(callchain[i]);
+        const char* current_method_name = findNativeMethod(callchain[i], &start_addr);
```

Comment:
`start_addr` seems to be ignored here. If that's the case, we should have a default value set to `nullptr` in `binarySearch`, and `binarySearch` should ignore the parameter if it's `nullptr`.